test/functional/test_framework/comptool.py

```diff
@@ -192,9 +192,7 @@ def disconnected():
         return wait_until(disconnected, timeout=10)
 
     def wait_for_verack(self):
-        def veracked():
-            return all(node.verack_received for node in self.test_nodes)
-        return wait_until(veracked, timeout=10)
+        [node.wait_for_verack() for node in self.test_nodes]
```

Comment:
Any reason not to return the value like this?

    return all(node.wait_for_verack() for node in self.test_nodes)